Apps/Sandcastle/gallery/3D Tiles.html

```diff
@@ -29,35 +29,54 @@
     "use strict";
 //Sandcastle_Begin
 
-var viewer = new Cesium.Viewer('cesiumContainer');
-var scene = viewer.scene;
-var city = scene.primitives.add(new Cesium.Cesium3DTileset({
-    //url : 'http://localhost:8002/tilesets/Cambridge',
-    //url : 'http://localhost:8002/tilesets/London_Canary_Wharf',
-    //url : 'http://localhost:8002/tilesets/trees',
-    //url : 'http://localhost:8002/tilesets/Philly',
-    url : 'http://localhost:8002/tilesets/Seattle',
-    //url : 'http://localhost:8002/tilesets/SeattleTrees',
-    //url : 'http://localhost:8002/tilesets/SeattleComposite',
-
-    // Defaults:
-    maximumScreenSpaceError : 16,
-    debugShowStatistics : false,
-    debugFreezeFrame : false,
-    debugShowBoundingVolume : false,
-    debugShowContentBoundingVolume : false
-}));
-
-viewer.camera.setView({
-    //destination : Cesium.Cartesian3.fromDegrees(-71.1106, 42.3736, 5000.0) // Cambridge
-    //destination : Cesium.Cartesian3.fromDegrees(0.0183, 51.5036, 5000.0) // Canary Wharf
-    //destination : Cesium.Cartesian3.fromRadians(-1.2911323805815227, 0.7097150757974291, 5000.0) // NYC Trees
-    //destination : Cesium.Cartesian3.fromDegrees(-75.1667, 39.9500, 5000.0) // Philly trees
-    destination : Cesium.Cartesian3.fromDegrees(-122.3331, 47.6097, 5000.0) // Seattle
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    scene3DOnly : true
 });
 
+var scene = viewer.scene;
+scene.fog.enabled = false;
 scene.debugShowFramesPerSecond = true;
-//viewer.extend(Cesium.viewerCesiumInspectorMixin);
+
+var tilesetUrl = 'http://localhost:8080/Specs/Data/Cesium3DTiles/Tilesets/Tileset/';
```

Comment:
These urls should not be hardcoded and they should not be absolutely.  They need to be relative in order to work in all cases and on the Cesium website.